packages/providers/src/utils/dumpContext.test.ts

```
@@ -9,6 +9,26 @@ import { promises as fs } from 'node:fs';
 import * as path from 'node:path';
 import * as os from 'node:os';
 import { dumpContext, redactSensitiveData, shouldDump } from './dumpContext.js';
+import { dumpContext, redactSensitiveData, shouldDump } from './dumpContext.js';
```

Biome (2.5.0)

[error] 'dumpContext' is redeclared in the same scope.

(lint/suspicious/noRedeclare)